tools/local-network/local_network.py

```diff
@@ -69,11 +69,13 @@ def start(self, network):
         if LOGSTASH_HOST:
             self.start_filebeat(LOG_BRANCH, LOGSTASH_HOST)
 
-        if GRAFANA_PASSWORD:
-            hosts = ', '.join(
-                f"'127.0.0.1:{BASE_ADMIN_HTTP_GATEWAY_PORT + i}'" for i in range(len(network.nodes))
-            )
-            self.start_prometheus(LOG_BRANCH, GRAFANA_PASSWORD, hosts)
+        if not GRAFANA_PASSWORD:
```

Comment:
This might not be new functionality, so feel free to ignore, but I'm confused on why its GRAFANA_PASSWORD?   We don't push logs to grafana?

Reply:
its actually api key i guess, so that prometheus can push to grafana from the local network.

this GRAFANA stuff in local_network.py dates to like year 2 of mobilecoin inc., i dont think its been used in a really long time